### ISSUE TITLE
[VRM - Bug]: Scheduling fixed prices not only changes per hour

### ISSUE THREAD

**Issue by dirkjanfaber** [victronenergy/dynamic-ess#84]
### Country / region

None

### feed-in possible

None

### feed-in control

None

### What happened?

> The DESS setup page in the beta VRM portal only seem to allow me setup times using hours, not minutes. So for example I can't set it up to have fixed pricing between 23:30 and 05:30, I have to round up/ down to 00:00 and 05:00.

Reported via https://community.victronenergy.com/articles/232720/dynamic-ess-on-beta-vrm.html?childToView=233170#comment-233170 by _sieade245_

### Screenshots

```bash
![DESCRIPTION](LINK.png)
```

**Comment by Stroller1024** [comment]
@dirkjanfaber This is correct.  UK requirements for 'Smart Meters', are that they can see the energy used over every half-hour period.

**Comment by ollyro** [comment]
I was about to log the same ticket then found this one. I'm also in the UK. Same issue when setting up a fixed prices schedule. I'm not aware of any UK tariff or pricing scheme with a granularity of less then 30 mins.

**Comment by BigBassettHound** [comment]
For CZ it would be nice to have option to add interval by minutes as low/high price is changing based on defined minutes intervals ie: High - 13:16-14:10. 
Another option is to read times from distributors api - but I haven't study it in detail yet.

**Comment by ajnadox** [comment]
I would also like to possibility to have by minutes, Sweden and most of the EU will from 2025 start using 15 min reading instead of reading usage per hour.
So a change to this will be needed.

The EU info meeting about this https://www.entsoe.eu/network_codes/cacm/implementation/mcsc/#market-coupling-consultative-group-mccg

**Comment by CVKBaca** [comment]
Many European utilities use time-of-use tariffs with switching times not aligned to 30-minute blocks (e.g., 00:15, 06:30, 07:30, 08:45, 09:15, 10:15, 11:15, 21:30, 22:00, 23:15).
With only 30-minute scheduling, Dynamic ESS cannot perfectly match these tariff changes, causing charging during short high-price windows or missing short low-price opportunities.

Allowing at least 15-minute intervals would enable precise alignment with real tariff schedules, reducing costs and maximizing low-price periods.

Example – Slovakia (ZSE, DD5 tariff with HDO control):
In my case, my low-tariff and high-tariff periods are defined by HDO signals from my electricity provider, ZSE. The times are not on the half-hour, which makes current scheduling options insufficient. Here is my real schedule:

Low tariff (€0.134376/kWh): 00:15–06:30, 07:30–08:45, 09:15–10:15, 11:15–21:30, 22:00–23:15

High tariff (€0.172813/kWh): all other times

A screenshot of my actual HDO schedule is attached below for reference - https://www.zsdis.sk/Uvod/Online-sluzby/Casy-prepinania-nizkej-a-vysokej-tarify

<img width="1044" height="1067" alt="Image" src="https://github.com/user-attachments/assets/85571ac7-b6ad-45e8-8b9f-96f5d8cc06d1" />
